unittests/eosio.token_tests.cpp

```diff
@@ -177,7 +177,7 @@ BOOST_FIXTURE_TEST_CASE( create_max_decimals, eosio_token_tester ) try {
    share_type amount = 0x8ac7230489e80000L;
    static_assert(sizeof(share_type) <= sizeof(asset), "asset changed so test is no longer valid");
    static_assert(std::is_trivially_copyable<asset>::value, "asset is not trivially copyable");
-   memcpy(&max, &amount, sizeof(share_type)); // hack in an invalid amount
+   memcpy((char*)&max, &amount, sizeof(share_type)); // hack in an invalid amount
```

Comment:
These were `copying an object of non-trivial type` warnings. Gnarly fix but, well, comment says it's a hack :wink:

Reply:
I removed this fix for now, so the other clearly correct ones can be fast pathed in